enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -173,27 +184,96 @@ Usage of the Thanos querier component inside cluster monitoring has many advanta
 
 See [5] and [6] for more details.
 
-### Tenancy
+#### Alerts/Recording Rules Aggregation
 
-![](user-monitoring-request.png)
+Thanos Querier only solves aggregating the query path. For aggregating recording rules as well as alerting rules
+another component [Thanos Ruler](https://thanos.io/components/rule.md/) is deployed. This component is responsible for hosting user defined recording rules
+as well as alerting rules.
+
+Thanos Ruler is deployed via the user workload monitoring prometheus operator via a new CRD called ThanosRule.
+The same _deny list_ setting is applied to user workload monitoring prometheus operator such that the user cannot
+deploy its own Thanos Ruler instances in user namespaces.
+
+Thanos Ruler queries Thanos Querier to resolve recording and alerting rules.
+This does have implications on the availability of user defined alerts.
+The following list enumerates critical components for alerts to work for this use case:
+
+1. Thanos Ruler
+2. Thanos Querier
+3. User workload Prometheus
+4. Cluster monitoring Prometheus
+
+If there is no need to aggregate user defined metrics and cluster level metrics in user alerts,
+higher availability can be achieved by hosting user defined alerting rules on the
+user workload Prometheus instances.
+
+This is enabled by setting a label on the PrometheusRule custom resource which is deployed by the user.
```

Comment:
can we be explicit what this label is going to be?

Reply:
Agreed, how about `openshift.io/cluster-monitoring-target: "prometheus"`? :thinking:

Reply:
I was more thinking something in the direction of: `openshift.io/evaluation-scope: "leaf-prometheus"`

Reply:
sounds good to me

Reply:
fixed